### PR TITLE
Update bitcoin-unlimited to 1.0.3.0

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/bitcoin-unlimited.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 cask 'bitcoin-unlimited' do
   version '1.0.2.0'
-  sha256 'e0360d108cb4f3ea996e5d59c41e95065e793c140bc12042e217d037bb587d63'
+  sha256 '41c977acb2dfa4d44c2dc8a11a7585330036af274b038ff0d5b592399491cf1d'
```
Comment (commitay) [inline Casks/bitcoin-unlimited.rb]: The checksum shouldn't change if this is a mirrored copy.

Comment (core-code) [reply Casks/bitcoin-unlimited.rb]: i was wondering about that as well, but the official download ( https://www.bitcoinunlimited.info/download ) now links there

**Comment by core-code** [comment]
ok but we have the same checksum mismatch as the original host.

the checksum failure on install is why i sent the pull request

**Comment by commitay** [comment]
Please update this to the current version `1.3.0.1` and keep the bitcoinunlimited.info URL.

**Comment by core-code** [comment]
i'm a bit hesitant to submit an pull request to 
• the newest version BUcash 1.3.0 because i don't know if its the same thing and also
• to the 1.0.3 version you mentioned, because while there seems to be a valid download there, its not mentioned at all on their download website

**Comment by commitay** [comment]
I posted a link to a different version by accident, the URL and version currently used in this PR is correct.

**Pull request merged**